views/layout.erb

```diff
@@ -12,19 +12,13 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link
-      href="https://unpkg.com/@umich-lib/css@v1/dist/umich-lib.css"
-      rel="stylesheet"
-      />
+
+    <link href="https://unpkg.com/@umich-lib/web@1/umich-lib.css" rel="stylesheet"/>
     <link href='/bundles/index.css' rel="stylesheet" />
-    <script
-      type="module"
-      src="https://unpkg.com/@umich-lib/components@v1/dist/umich-lib/umich-lib.esm.js"
-      ></script>
-    <script
-      nomodule
-      src="https://unpkg.com/@umich-lib/components@v1/dist/umich-lib/umich-lib.js"
-      ></script>
+
+    <script type="module" src="https://unpkg.com/@umich-lib/web@1/dist/umich-lib/umich-lib.esm.js"></script>
+    <script nomodule src="https://unpkg.com/@umich-lib/web@1/dist/umich-lib/umich-lib.js"></script>
```

Comment:
I can now see the updated components, which have reduced accessibility issues! 🎉